R/keep.R

```diff
@@ -16,8 +16,7 @@
 #'   * A named function, e.g. `is.character`.
 #'   * An anonymous function, e.g. `\(x) all(x < 0)` or `function(x) all(x < 0)`.
 #'   * A formula, e.g. `~ all(.x < 0)`. You must use `.x` to refer to the first
-#'     argument). Only recommended if you require backward compatibility with
-#'     older versions of R.
+#'     argument). No longer recommended.
```

Comment:
Here and below, I think it'd be helpful to be explicit about  why this is no longer recommended, e.g.:

```suggestion
#'     argument). No longer recommended as we now embrace the base R `\(x)` syntax added in R 4.1.
```